File: 01.JAVA/pg-166-169/pg_166_10.java
import java.util.Scanner;

public class pg_166_10
{
    public static void main(String[] args) {
        {
            Scanner enter = new Scanner(System.in);
            int num, dig1, dig2;

            System.out.println("");
            System.out.println("JVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJV");
            System.out.println("READ A TWO-DIGIT INTEGER AND DETERMINE IF BOTH DIGITS ARE THE SAME");
            System.out.println("JVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJV");
            System.out.println("");

            System.out.print("Write a two-digit integer: ");
            num = enter.nextInt();

            if(num < 0)
            {
                num *= -1;
            }

            if(num >= 10 && num <=99)
            {
                dig2 = num % 10;
                dig1 = num / 10;

                if(dig1 == dig2)
                {
                    System.out.println("Both digits are the same.");
                }
                else
                {
                    System.out.println("Both digits aren't the same.");
                }
            }
            else
                {
                    System.out.println("The written number doesn't have two digits. Please try again!");
                }
        }
    }
}

File: 03.JAVASCRIPT/pg-166-169/pg_166_10.js
console.log("");
console.log("JSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJS");
console.log("READ A TWO-DIGIT INTEGER AND DETERMINE IF BOTH DIGITS ARE THE SAME");
console.log("JSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJS");
console.log("");

let numInput = document.querySelector("#num");
let btnProof = document.querySelector(".btnProof");
let textar = document.querySelector("#textar");

let dig1, dig2;

function validateNumber()
{
    let num = Number(numInput.value);

    if(num < 0)
    {
        num *= -1;
    }

    if(num >= 10 && num <= 99)
    {
        dig2 = num % 10;
        dig1 = Math.floor(num/10);
        
        if(dig1 == dig2)
        {
            textar.value = "Both digits are the same."
        }
        else
        {
            textar.value = "Both digits aren't the same."
        }
    }
    else
    {
        textar.value = "The written number doesn't have two digits. Plese try again!";
    }
}

btnProof.addEventListener("click", validateNumber);
numInput.addEventListener("keydown", function(event)
{
    if(event.key === "Enter")
    {
        validateNumber();
    }
});